README.md

```diff
@@ -76,65 +76,65 @@ in remote mode (meaning that `lnd-mode=remote` is set). It shows the
 (LiT). The bundled version will always come with the correct,
 [compatible versioning](#daemon-versions-packaged-with-lit).
 
-| LiT                   | LND          |
-|-----------------------|--------------|
-| **v0.16.0-alpha.rc1** | v0.19.0-beta |
-| **v0.15.3-alpha**     | v0.19.0-beta |
-| **v0.15.2-alpha**     | v0.19.0-beta |
```

Comment:
IMO min LND version support should just be in the release notes on the GitHub release page. I think of this as release specific data and not really top level project README data. I think we should retire this table to `./docs` and just add a note to the GitHub release page (and template).